### ISSUE TITLE
License:  Finetune CHAI1 weights with DFT loss?

### ISSUE THREAD

**Issue by AlexanderMath** [chaidiscovery/chai-lab#143]
Want to finetune {openfold,alphafold,chai,esmfold} with [quantum chemistry](https://en.wikipedia.org/wiki/Density_functional_theory) labels.  Does the CHAI1 license permit this, or can I only do this for {openfold,alphafold,esmfold}? 

(openfold and alphafold is expected to become opensource soon)

**Comment by arogozhnikov** [comment]
Hi Alexander, 

fine-tuning requires permission, see https://github.com/chaidiscovery/chai-lab/issues/125#issuecomment-2430455069
